test/fixtures/v1StartsEndsTest.js

```diff
@@ -0,0 +1,122 @@
+const tree = [
+    {
+        'List': [
```

Comment:
Why do you need to use a `String` to define a `key` when it's not necessary?

Example:
```suggestion
        List: [
```